modelforge/potential/ani.py

```diff
@@ -97,11 +86,11 @@ class ANIRepresentation(nn.Module):
 
     def __init__(
         self,
-        radial_max_distance: unit.Quantity,
-        radial_min_distanc: unit.Quantity,
+        radial_max_distance: float,
+        radial_min_distance: float,
         number_of_radial_basis_functions: int,
-        maximum_interaction_radius_for_angular_features: unit.Quantity,
-        minimum_interaction_radius_for_angular_features: unit.Quantity,
+        maximum_interaction_radius_for_angular_features: float,
+        minimum_interaction_radius_for_angular_features: float,
```

Comment:
Since the angular features will be set based on the pair list we calculated from the radial_max_distance, do we need to add in some logic in our parameters model validation to ensure that the angular_features cutoff is not larger than the radial_max_distance?

Reply:
that's a good point! we should check for this in the pydantic model